src/plugins/data/common/field_formats/constants/base_formatters.ts

```diff
@@ -40,7 +39,6 @@ export const baseFormatters: FieldFormatInstanceType[] = [
   BoolFormat,
   BytesFormat,
   ColorFormat,
-  DateNanosFormat,
```

Comment:
`DateNanosFormat` should not be a "common" formatter, since it requires different server-side behavior. It needs this because of the possibility that `dateFormat:tz` setting could be `Browser`